packages/react-native-webrtc/README.md

```diff
@@ -39,48 +39,6 @@ After installing this npm package, add the [config plugin](https://docs.expo.io/
 
 Next, rebuild your app as described in the ["Adding custom native code"](https://docs.expo.io/workflow/customizing/) guide.
 
-### Event Target Shim
-
-> SDK 50 and greater.
```

Comment:
This is still relevant for SDK 50 users. Would just change it to not say "and greater"